swift/src/Ice/Communicator.swift

```diff
@@ -190,38 +179,36 @@ public protocol Communicator: AnyObject, Sendable {
     /// Ice.Admin.InstanceName is not set. If Ice.Admin.DelayCreation is 0 or not set, getAdmin is called
     /// by the communicator initialization, after initialization of all plugins.
     ///
-    /// - returns: `ObjectPrx?` - A proxy to the main ("") facet of the Admin object, or a null proxy if no Admin
-    /// object is configured.
+    /// - Returns: A proxy to the main ("") facet of the Admin object, or a null proxy if no Admin
```

Comment:
```suggestion
    /// - Returns: A proxy to the main ("") facet of the Admin object, or a nil proxy if no Admin
```

We use both null and nil for the comments, we have settle on one, I think nil is the more appropriate for Swift.